src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -84,11 +84,8 @@ class ReimbursementAccountPage extends React.Component {
         this.continue = this.continue.bind(this);
         this.getDefaultStateForField = this.getDefaultStateForField.bind(this);
         this.goBack = this.goBack.bind(this);
-        const achData = lodashGet(this.props.reimbursementAccount, 'achData', {});
-        const hasInProgressVBBA = achData.bankAccountID && achData.state !== BankAccount.STATE.OPEN && achData.state !== BankAccount.STATE.LOCKED;
-
         this.state = {
-            shouldHideContinueSetupButton: !hasInProgressVBBA,
+            shouldShowContinueSetupButton: this.getShouldShowContinueSetupButtonInitialValue(),
```

Comment:
Flipped the state to a positive sentence because I think it is easier to reason about it